website/vercel.json

```diff
@@ -0,0 +1,3 @@
+{
+  "version": 2
```

Comment:
If this is empty I think we can remove it entirely

Reply:
Yea, I kept it for now because we may be adding the "silent github" flag, but if not I'll remove the file.